native/packages/react-native-bpk-component-picker/src/BpkPicker-test.common.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import BpkPicker from './BpkPicker';
+import BpkPickerItem from './BpkPickerItem';
+
+const commonTests = () => {
+  describe('BpkPicker', () => {
+    const emptyFn = () => null;
```

Comment:
we should use `jest.fn()` and potentially check is those function have been called properly